app/docs/how-to/streamline-curbside.md

```diff
@@ -0,0 +1,255 @@
+
+Let's get started!
+
+### Creating the Routegy Touchpoint for curbside delivery
+
+We are going to start with creating a [touchpoint](/reference/touchpoints/) that collects desired information from the customer when they arrive and scan the code. We are going to ask how did they place the order, their name and some details about their car.
+
+#### Step 1: Choose workspace
+
+Start by making sure you're in the desired [workspace](/reference/workspaces/). Click on the menu bar dropdown to choose, and then select **Touchpoints** tab.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/workspace-touchpoints.png" width="90%">
+</p>
+
+#### Step 2: Create a new location
+
+Next step is to create a new [location](/reference/locations/) for our curbside touchpoint. If you already have a location that you'd like to use, you can skip this step and go directly to [Step 3](#step-3-create-a-new-touchpoint).
+
+Create a new location by dragging a location icon into a desired parent location.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/location-drag-drop.png" width="90%">
+</p>
+
+Then, give your new location a name, description (optional) and click **Create Location** button.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/location-dialog.png" width="75%">
+</p>
+
+#### Step 3: Create a new touchpoint
+
+Create a new touchpoint by dragging a touchpoint icon into a desired location. In this example, we are putting it under the *Parking* location created in [Step 2](#step-2-create-a-new-location).
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/touchpoint-drag-drop.png" width="90%">
+</p>
+
+Then, define name and description for your new touchpoint.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/touchpoint-name-description.png" width="75%">
+</p>
+
+Routegy will use the name and description in the header of your new touchpoint form as its title and subtitle respectively (see screenshot below). With this in mind, try using short, concise language to provide clear instructions to your customers (e.g. a simple call to to action).
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/curbside-form-name-description.png" width="30%">
+</p>
+
+#### Step 4: Define touchpoint schema
+
+Next, we are going to define a touchpoint [schema](/reference/schemas/). Schema is an exteremely important part of a touchpoint that defines what kind of data is collected from people interacting with your touchpoint.
+
+To get started, select **Define your own schema** from **Schema** dropdown list and name your new schema.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/touchpoint-define-schema.png" width="75%">
+</p>
+
+The Routegy schema is defined by a [JSON Schema](https://json-schema.org/) document in YAML format. To collect customer's name, car information and order type, we are going to use the following schema:
```

Comment:
"The Routegy schema is defined..." -> "Routegy schemas are defined..."

Reply:
"car information and order type" -> "car information, and order type" (missing comma i believe)